sdk/documentintelligence/azure-ai-documentintelligence/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,151 @@
+
+To enable a wider set of scenarios, this SDK introduces a keyword argument "split" to specify the document splitting mode with the new "2023-10-31-preview" service version. The following values are supported:
+
+- `split: "auto"`
+
+Let service determine where to split.
+
+- `split: "none"`
+
+The entire file is treated as a single document. No splitting is performed.
+
+- `split: "perPage"`
+
+Each page is treated as a separate document. Each empty page is kept as its own document.
+
+## Breaking Changes
+
+### Clients names updates.
+
+|**2023-07-31**|**2023-10-31-preview**|
+|----------------|-------------|
+|DocumentAnalysisClient|DocumentIntelligenceClient|
+|DocumentModelAdministrationClient|DocumentIntelligenceAdministrationClient|
+
+### Base endpoint updates 
+Updates all REST API operation paths from `{endpoint}/formrecognizer` to `{endpoint}/documentintelligence`. SDK would handle this change automatically, users would not have to do additional work to support this.
+
+### Not backword compatible with azure-ai-formrecognizer
+`azure-ai-documentintelligence` is a new package, it is not compatible with the previous `azure-ai-formrecognizer` package without necessary changes to your code.
+
+### API shapes changes
+API shapes have been designed from scratch to support the new Client for the Document Intelligence service. Please refer to the [Readme](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/README.md) and [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/documentintelligence/azure-ai-documentintelligence/samples) for more understanding.
+
+### Fields changes in prebuilt-receipt model
+In `prebuilt-receipt` model, change to tract currency-related fields values from _number_ to _currency_.
+
+```json
+"Total": {
+    "type": "currency",
+    "valueCurrency": {
+        "amount": 123.45,
+        "currencySymbol": "$",
+        "currencyCode": "USD"
+    },
+    ...
+}
+```
+Now each currency-related field returning its own currency info to better support receipts with multi-currency, so the _Currency_ field in result has been removed.
+
+```json
+"fields": {
+    "Total": {
+        "type": "currency",
+        "valueCurrency": {
+            "amount": 123.45,
+            "currencySymbol": "$",
+            "currencyCode": "USD"
+        },
+    ...
+    },
+    "Tax": { "type": "currency", "valueCurrency": ... },
+    ...
+}
+```
+
+### Model Retirements/Deprecations.
+
+- `"prebuilt-businessCard"` model is retired.
+
+- `"prebuilt-document"` model is retired, this model is essentially `"prebuilt-layout"` with `features="keyValuePairs"` specified. _(This is only supported as an optional feature for "prebuilt-layout" and "prebuilt-invoice".)_
+
+If you wish to still use these models, please rely on the older `azure-ai-formrecognizer` library through the older service API versions.
```

Comment:
For this section you should link to the DI supported models table and guide people to refer to that table to get an up to date view of supported models

Reply:
Agree, I was also thinking about it but I didn't find the public link for DI supported models. It probably same like other DI links, not ready yet. We should link it here in future.